src/ameba/rule/naming/ascii_identifiers.cr

```diff
@@ -0,0 +1,68 @@
+module Ameba::Rule::Naming
+  # A rule that reports non-ascii characters in identifiers.
+  #
+  # Favour this:
+  #
+  # ```
+  # class BigAwesomeWolf
+  # end
+  # ```
+  #
+  # Over this:
+  #
+  # ```
+  # class BigAwesome🐺
+  # end
+  # ```
+  #
+  # YAML configuration example:
+  #
+  # ```
+  # Naming/AsciiIdentifiers:
+  #   Enabled: true
+  # ```
+  class AsciiIdentifiers < Base
+    include AST::Util
+
+    properties do
+      description "Disallows non-ascii characters in identifiers"
+    end
+
+    MSG = "Identifier contains non-ascii characters"
+
+    def test(source, node : Crystal::Assign)
```

Comment:
there is also [MultiAssign](https://github.com/crystal-lang/crystal/blob/master/src/compiler/crystal/syntax/ast.cr#L875)

Reply:
Multiple assignment is [not allowed](https://github.com/crystal-lang/crystal/blob/c25640d79bcbf16b015cf9439a862d4626973cf2/src/compiler/crystal/syntax/parser.cr#L150) for constants.

Reply:
Actually, I'll extend these checks to all identifiers, not only constants.